lib/openai/client.rb

```diff
@@ -7,8 +7,8 @@ def initialize(access_token: nil, organization_id: nil, uri_base: nil, request_t
       OpenAI.configuration.request_timeout = request_timeout if request_timeout
     end
 
-    def chat(parameters: {})
-      OpenAI::Client.json_post(path: "/chat/completions", parameters: parameters)
+    def chat(deployment_id: nil, parameters: {})
+      OpenAI::Client.json_post(deployment_id: deployment_id, path: "/chat/completions", parameters: parameters)
```

Comment:
I'm surprised rubocop (lint) doesn't want to break up this line, like it wanted me to do with the others